### ISSUE TITLE
mzR fails to build on bioc devel windows (hdf5 related)

### ISSUE THREAD

**Issue by sgibb** [sneumann/mzR#143]
Since some time `mzR` fails to build from source on windows and bioc devel because of a missing hdf5 cpp header file ([build report](https://bioconductor.org/checkResults/devel/bioc-LATEST/mzR/tokay2-install.html)):
```
...
In file included from pwiz/data/msdata/mz5/Configuration_mz5.hpp:27:0,
                 from pwiz/data/msdata/mz5/Configuration_mz5.cpp:24:
pwiz/data/msdata/mz5/Datastructures_mz5.hpp:30:19: fatal error: H5Cpp.h: No such file or directory
 #include "H5Cpp.h"
                   ^
compilation terminated.
make: *** [pwiz/data/msdata/mz5/Configuration_mz5.o] Error 1
ERROR: compilation failed for package 'mzR'
```

The same error was present on Travis and my local machine which could be solved by adding libhdf5-dev as new dependency: https://github.com/sneumann/mzR/blob/4c1f1d63054c44087eeb9d23db6ef8c01427a736/.travis.yml#L15

I guess this problem is related to the (re)introduction of mz5 (https://github.com/sneumann/mzR/commit/d4910ec36d01a218baf948819c0e01827add8c92; I don't want to start the discussion of #120 again; IMHO this code is merged so the PR could be closed anyway; the "automatic detection of HDF5" would be a different problem).

Is there a way to fix the build on bioc devel's windows machines? E.g. adding it to `SystemRequirements` in `DESCRIPTION` and/or inform the bioc build service maintainers that we need HDF5 cpp for windows?

**Comment by sneumann** [comment]
Hi, 
following the suggestion by Martin Morgan on https://stat.ethz.ch/pipermail/bioc-devel/2018-January/012576.html I have started the branch https://github.com/sneumann/mzR/tree/feature/LinkToRhdf5lib
Note: since pwiz is including `#include "H5Cpp.h"`rather than `#include "c++/H5Cpp.h"` we need to find a fix for https://github.com/grimbough/Rhdf5lib/issues/5
Locally I have worked around that by `mv -i  c++/H5* .` inside `/usr/local/lib/R/site-library/Rhdf5lib/include`.
Yours, Steffen

**Comment by sneumann** [comment]
With some more workarounds on the travis side, I now have a successful build at https://travis-ci.org/sneumann/mzR/builds/324953759

**Comment by sneumann** [comment]
Fixed now in 2.13.3
